docs/podman-run.1.md

```diff
@@ -212,7 +212,7 @@ Limit write rate (IO per second) to a device (e.g. --device-write-iops=/dev/sda:
 
 **--dns**=*dns*
 
-Set custom DNS servers
+Set custom DNS servers. Invalid if using **--dns** with **--network** that is not set to host.
```

Comment:
I think this is 
Invalid if **--network** is set to host.
Is it invalid also if --network is set to a different container?
Is it invalid if --network is set to None?